easybuild/framework/easyconfig/types.py

```diff
@@ -33,10 +33,10 @@
 
 from easybuild.tools.build_log import EasyBuildError
 
-
 _log = fancylogger.getLogger('easyconfig.types', fname=False)
 
 
+
```

Comment:
need MOAR whitespace? ;-)

two empty lines above/below `_log` is ok, leave it as is